infrastructure/main.tf

```diff
@@ -131,16 +131,26 @@ resource "aws_iam_policy" "ecs_ssm_task_policy" {
       ],
       "Effect": "Allow",
       "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "s3:ListBucket",
+      "Resource": "arn:aws:s3:::*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": ["s3:GetObject", "s3:PutObject"],
+      "Resource": "arn:aws:s3:::*"
     }
   ]
 }
 EOF
 }
 
-resource "aws_iam_policy_attachment" "ecs_ssm_task_atttachment" {
+resource "aws_iam_policy_attachment" "ecs_task_atttachment" {
```

Comment:
what was the change here?

Reply:
the iam policy now includes ssm permissions and also s3 permissions, so I named it more abstractly